client/app/cloud/project/compute/infrastructure/virtualMachine/add/cloud-project-compute-infrastructure-virtualMachine-add.controller.js

```diff
@@ -91,12 +90,15 @@ class CloudProjectComputeInfrastructureVirtualMachineAddCtrl {
     this.addVirtualMachine();
   }
 
-  /*----------------------------------
-     |  Step 1 : OS or SnapShot choice  |
-     ----------------------------------*/
+  /*-----------------------------------------
```

Comment:
Do you think you could export Step1 and Step2 in different files ? It would enhance readability